.gitignore

```diff
@@ -7,6 +7,7 @@
 # Generated files
 .docusaurus
 .cache-loader
+yarn.lock
```

Comment:
Revert this change please.

Reply:
How to undo the change? I use ctrl+z, but can't

Reply:
Checkout the file you want from the commit on the `main` branch:

`git checkout main .gitignore`
`git checkout main yarn.lock`

Reply:
![1637347932(1)](https://user-images.githubusercontent.com/55161917/142675994-9a10aea3-1d3d-4831-b2e1-b6df4bc5cfac.png)
I also need to ` git checkout main docs/A-Introduction/information.md `
       and  ` git checkout main static/img/ram.pgn ` , right?

Reply:
then only add information.md and ram.pgn, right?

Reply:
From the beginning of the project. I create a new branch isssue-44, then changed information. md,  ram.pgn, .gitignore and yarn.block. then add commit all the four files. The last step is ` git push origin issue-44 `.

Reply:
I didn't merge the issue-44 to the main. Should I use this command ` git checkout issue-44 yarn.lock `?

Reply:
Seems like you have `.gitignore` and `yarn.lock` staged in git. Use `restore --staged <file>` to unstage them, use `git checkout issue044 <file>` to rollback their content. Then use `git add <file>` for files that you are meant to change, in this case only `docs/A-Introduction/information.md` and maybe files in `static/img`. Rebase it and force push.